docs/content.zh/docs/deployment/metric_reporters.md

```diff
@@ -24,31 +24,30 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# Metric Reporters
+# 指标发送器
```

Comment:
```suggestion
<a name="metric-reporters"></a>

# 指标发送器
```

Please fill the related tags in the rest of the section titles in the page.

Reply:
Ok,I will add tags later:)

Reply:
Tags are added.